src/components/RegistrationListItem/components/Title.tsx

```diff
@@ -0,0 +1,16 @@
+import { Typography } from '@mui/material';
+import { getTitleString } from '../../../utils/registration-helpers';
+import { RegistrationListItemContext } from '../context';
+import { useContext } from 'react';
+
+export const Title = () => {
+  const { registration } = useContext(RegistrationListItemContext) ?? {};
+  if (!registration) return null;
+  const title = registration?.mainTitle ?? '';
+
+  return (
+    <Typography gutterBottom sx={{ fontSize: '1rem', fontWeight: '600', wordBreak: 'break-word' }}>
+      {getTitleString(title)}
```

Comment:
noen spesiell grunn til at du ikke sender inn `registration.mainTitle` direkte i funksjonen, men lager en ny const for det?